responsibleai/responsibleai/modelanalysis/model_analysis.py

```diff
@@ -156,8 +160,9 @@ def _get_classes(task_type, train, target_column, train_labels):
 
     def _validate_model_analysis_input_parameters(
             self, model, train, test, target_column,
-            task_type, categorical_features=None, train_labels=None,
-            serializer=None):
+            task_type, categorical_features, train_labels,
```

Comment:
Why have you changed the optional parameters to this function?

Reply:
Because it's internal and only called from the constructor (which already has the defaults making them pointless here). Having defaults just makes it harder to add arguments (adding on the end means they need defaults, adding them earlier would change the order from the constructor).